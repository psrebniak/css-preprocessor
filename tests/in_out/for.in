$loop: test;

@for($loop; 1; 3) {
    @for($loop2; 2; 1) {
        .grid-${loop}-${loop2} {
            test: test;
        }
    }
}

.test {
    loop: ${loop};
}
