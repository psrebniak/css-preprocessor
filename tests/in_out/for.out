.grid-1-2 {
    test: test;
}
.grid-1-1 {
    test: test;
}
.grid-2-2 {
    test: test;
}
.grid-2-1 {
    test: test;
}
.grid-3-2 {
    test: test;
}
.grid-3-1 {
    test: test;
}
.test {
    loop: test;
}
